rebasehelper/helpers/macro_helper.py

```diff
@@ -41,6 +41,40 @@ def expand(s, default=None):
         except rpm.error:
             return default
 
+    @staticmethod
+    def expand_macros(macros):
+        """Expands values of multiple macros.
+
+        Args:
+            macros (list): List of macros to be expanded, macros are
+                represented as dicts.
+
+        Returns:
+            list: List of macros with expanded values.
+
+        """
+        for macro in macros:
+            macro['value'] = MacroHelper.expand(macro['value'])
+        return macros
+
+    @staticmethod
+    def substitute_path_for_macro(path, macros):
```

Comment:
Why is the method called like this? Shouldn't it rather be `substitute_path_with_macros`?